pypeit/core/wavecal/spectrographs/templ_gemini_gmos.py

```diff
@@ -11,18 +11,29 @@ def gemini_gmos_r400_hama(overwrite=False):  # GMOS R400 Hamamatsu
     binspec = 2
```

Comment:
we've been doing this kind of development in the Dev-Suite. I agree it should probably be done here but should we make a decision and migrate?

Reply:
I'm not sure what you mean. 
In the DevSuite are the files for
this and not the code.